doc/whatsnew/2.13.rst

```diff
@@ -77,6 +77,12 @@ Extensions
 Other Changes
 =============
 
+* Add ``--recursive`` option to allow linting recursively all modules and packages in subtree. Running pylint with
+  ``--recursive`` option will check all discovered ``.py`` files and packages found inside subtree of directory provided
+  as parameter to pylint.
+
+  Closes #352
```

Comment:
This should be copied to the `Changelog` file. ie., `2.13.rst` and `Changelog` should include the same changes.

Reply:
done